pkgs/top-level/all-packages.nix

```diff
@@ -16740,6 +16740,9 @@ in
 
   pkgdiff = callPackage ../tools/misc/pkgdiff { };
 
+  place-cursor-at =
+    lib.getBin (haskell.lib.justStaticExecutables haskellPackages.place-cursor-at);
```

Comment:
I think `getBin` is redundant since the argument does not appear to be a split derivation.

Reply:
@lourkeur Yes, you’re probably right, I just copy-pasted it from `niv`. Removed that.